adaptive_shield/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "manifest_version": "2.0.0",
+    "app_id": "adaptive-shield",
+    "app_uuid": "",
+    "display_on_public_website": true,
+    "tile":
+    {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Track SaaS posture alerts",
+        "title": "Adaptive Shield",
+        "media":
+        [],
+        "classifier_tags":
+        [
+            "Supported OS::Linux",
+            "Supported OS::Mac OS",
+            "Supported OS::Windows",
+            "Category::Alerting",
+            "Category::Cloud",
+            "Category::Security",
+            "Offering::Integration"
+        ]
+    },
+    "author":
+    {
+        "homepage": "https://www.adaptive-shield.com/",
+        "name": "Adaptive Shield",
+        "support_email": "support@adaptive-shield.com",
+        "sales_email": ""
```

Comment:
Took a look at the validations and it looks like this is a required field. Would you be able to add a sales email from your end here? If there's no specific email for sales, the support email would work fine.